price_transparency/hospitals/machine_readable_links.csv

```diff
@@ -1,184 +1,365 @@
+,Uniontown Hospital,hospital,https://wvumedicine.org/wp-content/uploads/2022/03/250965588_Uniontown_standardcharges.xlsx,up,https://wvumedicine.org/bill-pay/pricing-transparency/,,,xlsx,FALSE,"File type is xlsx. File naming convention not followed. File labeled as shoppable services, perhaps not even intended to be MRF?",PA,,2022-06-08,
+,Wetzel County Hospital,hospital,https://wvumedicine.org/wp-content/uploads/2022/03/843480493_WetzelCountyHospital_standardcharges.xlsx,up,https://wvumedicine.org/bill-pay/pricing-transparency/,,,xlsx,FALSE,"File type is xlsx. File naming convention not followed. File labeled as shoppable services, perhaps not even intended to be MRF?",WV,,2022-06-08,
+,Wheeling Hospital,hospital,https://wvumedicine.org/wp-content/uploads/2022/02/550357057_wheeling-hospital_standardcharges-1.xlsx,up,https://wvumedicine.org/bill-pay/pricing-transparency/,,,xlsx,FALSE,"File type is xlsx. File naming convention not followed. File labeled as shoppable services, perhaps not even intended to be MRF?",WV,,2022-06-08,
+Adventist HealthCare,Adventist HealthCare Fort Washington Medical Center,hospital,https://www.adventisthealthcare.com/app/files/public/91919297-0df2-4ab4-b788-71160672291c/FWMC-Billing-StandardCharges.csv,up,https://www.adventisthealthcare.com/patients-visitors/billing-financial/price-transparency/,,,csv,,,MD,2021-12-21,2022-06-08,Not to be confused with Adventist Health
+,Adventist HealthCare Shady Grove Medical Center,hospital,https://www.adventisthealthcare.com/app/files/public/5234573e-a91f-456f-919e-dd37620e586c/SGMC-Billing-StandardCharges.csv,up,https://www.adventisthealthcare.com/patients-visitors/billing-financial/price-transparency/,,,csv,,,MD,2021-12-21,2022-06-08,
+,Adventist HealthCare White Oak Medical Center,hospital,https://www.adventisthealthcare.com/app/files/public/d73a12fc-fecc-4674-9a62-03326b134182/WOMC-Billing-StandardCharges.csv,up,https://www.adventisthealthcare.com/patients-visitors/billing-financial/price-transparency/,,,csv,,,MD,2021-12-21,2022-06-08,
+,Adventist HealthCare Rehabilitation,hospital,https://www.adventisthealthcare.com/app/files/public/90def31e-c6c7-43b7-b30f-6eb68b9d2844/REHAB-Billing-StandardCharges.csv,up,https://www.adventisthealthcare.com/patients-visitors/billing-financial/price-transparency/,,,csv,,,MD,2021-12-21,2022-06-08,
+CalvertHealth System,CalvertHealth Medical Center,hospital,https://www.calverthealthmedicine.org/price-transparency-file,up,https://www.calverthealthmedicine.org/Charge-Master-Dictionary-Price-Transparency,,,xlsx,FALSE,"File type is xlsx. File naming convention not followed. File labeled as shoppable services, perhaps not even intended to be MRF?",MD,,2022-06-08,
+ChristianaCare,"ChristianaCare, Union Hospital",hospital,???,up,https://billing.christianacare.org/transparency,,,,,,MD,,2022-06-08,Confusing; seems like there's no machine-readable file

```

Comment:
We should think about what the most useful way to record information for MRFs we are unable to locate is. Technically I think this row does meet the current schema requirements, but the schema is not well-designed for this case. For now, I think since you did not find an MRF url (and entered `???` as a placeholder), you should switch the `machine_readable_url_status` from `up` to `down`.

Longer term, I think the schema should be modified to handle such cases more naturally. One idea is we could change the `machine_readable_url` to be an optional field, and change the `machine_readable_url_status` to be an enum including `up`, `down` and `not_found`. Then we can provide working urls, purportedly working urls, and metadata for hospitals for which we couldn't find urls all in the same space, in a way that allows users to easily filter for working urls.